src/InvoiceGenerator/InvoiceGenerator.BusinessLogic/ClientViewModel.cs

```diff
@@ -1,10 +1,8 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 
-namespace InvoiceGenerator.Models
+namespace InvoiceGenerator.BusinessLogic
 {
     public class ClientViewModel
```

Comment:
We'll move these ViewModels to a ViewModels class library soon, and reference them from both the UI and BLL. Not something we have to do right now, as it's a clean up task - already added to the Project.